teams/3/list-examples.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="./styles.css" />
+        <link rel="stylesheet" href="../../css/normalize.css" />
+        <link rel="stylesheet" href="../../css/main.css" />
+        <link rel="stylesheet" href="../../css/vertical-navigation.css" />
+        <title>Document</title>
+    </head>
+    <body>
+        <h1>List examples</h1>
+        <section>
+            <h2>Example number 1</h2>
+            <ul class="ut-list-cols-2">
+                <li class="text-xl-med">
+                    <img src="./assets/clock.svg" alt="clock" />
+                    <h3>Access to daily analytics</h3>
+                    <p>
+                        Optimize the way you recover, train, and sleep with
+                        daily reporting on mobile and desktop apps. Start
+                        training smarter, not harder.
+                    </p>
+                </li>
+                <li class="text-xl-med">
+                    <img src="./assets//charge.svg" alt="charge" />
+                    <h3>Measure recovery</h3>
```

Comment:
This elements should be span, please review that all the list content have the corresponding text styles

Reply:
hello grace, thanks we are committing the changes shortly

Reply:
The same reason as I have written above.